cluster/gce/gci/configure-helper.sh

```diff
@@ -900,6 +900,9 @@ function start-kube-apiserver {
 
 
   local authorization_mode="RBAC"
+  # Load existing ABAC policy files written by versions < 1.6 of this script
+  # TODO: only default to this legacy path when in upgrade mode
+  ABAC_AUTHZ_FILE="${ABAC_AUTHZ_FILE:-/etc/srv/kubernetes/abac-authz-policy.jsonl}"
```

Comment:
The container-linux script was recently renamed from "coreos". It was heavily refactored after the 1.5 cut (due to my slow review) and I'm not sure if the maintainers are concerned with upgrade compatibility as they are still heavily modifying the script.